lib/internal/webstreams/readablestream.js

```diff
@@ -702,7 +699,7 @@ class ReadableStreamBYOBRequest {
     const viewBuffer = ArrayBufferViewGetBuffer(view);
     const viewBufferByteLength = ArrayBufferPrototypeGetByteLength(viewBuffer);
 
-    if (isArrayBufferDetached(viewBuffer)) {
+    if (viewBuffer.detached) {
```

Comment:
this is a mutable property, so it must be a primordial
```suggestion
    if (ArrayBufferPrototypeDetached(viewBuffer)) {
```